From 2b8f4535149571c85ee6db248c1eeeabe38d044a Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Sat, 10 Oct 2009 16:12:41 +0800
Subject: [PATCH] Replace Idle function with standard VBI interface

Since vbi_ctx_ctl isn't the standard VBI 2.0 interface, so replace
it with vbi_idle() in power_save function.

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 arch/powerpc/kernel/vbi/wrhv.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/kernel/vbi/wrhv.c b/arch/powerpc/kernel/vbi/wrhv.c
index ad9d892..b78ae97 100644
--- a/arch/powerpc/kernel/vbi/wrhv.c
+++ b/arch/powerpc/kernel/vbi/wrhv.c
@@ -175,7 +175,7 @@ unsigned long __init wrhv_find_end_of_memory(void)
 void wrhv_power_save(void)
 {
 	local_irq_enable();
-	vbi_ctx_ctl(VBI_CTXCTL_IDLE, 1, 0);
+	vbi_idle(1);
 }
 
 void wrhv_restart(void)
-- 
1.6.5.2

